openapi/whohoststhis/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to [Who Hosts This](https://www.who-hosts-this.com/Documentation) from Ballerina
+
+## Package overview
+The `ballerinax/whohoststhis` is a [Ballerina](https://ballerina.io/) connector for Who Hosts This API.
+This package provides the capability to discover the hosting provider for any web site.
+
+### Compatibility
+|                    | Version                   |
+|--------------------|---------------------------|
+| Ballerina Language | Ballerina Swan Lake Beta3 | 
+| Who Hosts This API | v0.0.1                    |
+
+## Report issues
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina connector repository](https://github.com/ballerina-platform/ballerinax-openapi-connectors)
```

Comment:
Give link to ballerina-platform/ballerina-extended-library

Reply:
Added that in Package.md